cli/src/main/java/io/github/nubesgen/cli/subcommand/ScanCommand.java

```diff
@@ -70,6 +70,9 @@ public static String scan(String workingDirectory) {
                 } else if (testFile.contains("io.quarkus")) {
                     Output.printInfo("Runtime selected: Quarkus + Maven");
                     getRequest += "&runtime=QUARKUS";
+                } else if (testFile.contains("micronaut")) {
```

Comment:
Other framework seem to be using a package name `io.quarkus`, `org.springframework.boot`? Should we do that?

Reply:
We cannot guarantee `io.micronaut` is in the build.  They could be using a version catalog...